### DISCUSSION TITLE
Git账户的用户名和密码

### DISCUSSION THREAD

**Discussion by QKQS** [community/community#122957] [New to GitHub]
git账户注册登录好了但长时间没使用，用户名和密码给忘了，百度说在GitHub上面能够重置，应该怎么样进行重置密码？我到现在为止还没有找到办法进行git仓库的代码使用，请求大佬给点意见和建议。

**Comment by ghostinhershell** [comment]
Hi @QKQS, thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
